src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/DecisionAndReasonsStartedConfirmation.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.handlers.postsubmit;
+
+import static java.util.Objects.requireNonNull;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.Event;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.Callback;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PostSubmitCallbackResponse;
+import uk.gov.hmcts.reform.iacaseapi.domain.handlers.PostSubmitCallbackHandler;
+
+@Component
+public class DecisionAndReasonsStartedConfirmation implements PostSubmitCallbackHandler<AsylumCase> {
+
+    public boolean canHandle(
+        Callback<AsylumCase> callback
+    ) {
+        requireNonNull(callback, "callback must not be null");
+
+        return callback.getEvent() == Event.DECISION_AND_REASONS_STARTED;
```

Comment:
Do we need to check the phase here?  E.g. `&& callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT`

Reply:
The other handlers dont, so I'm not sure we do

Reply:
Isn't the stage defined by the abstract handler that it extends?

Reply:
Since this is a `PostSubmitCallback` I think you're right.  The phases only really apply to `PreSubmitCallback` since you can have aboutToStart or aboutToSubmit.